docs/sources/flow/reference/cli/run.md

```diff
@@ -52,3 +55,19 @@ All components managed by the component controller are reevaluated after
 reloading.
 
 [component controller]: {{< relref "../../concepts/component_controller.md" >}}
+
+## Clustered mode (experimental)
+
+When the `--cluster.enabled` command-line argument is provided, Grafana Agent will
+start in _clustered mode_.
+
+The agent tries to connect over HTTP/2 to one or more peers provided in the
+comma-separated `--cluster.join-address` list to join an existing cluster.
+If no connection can be made or the argument is empty, the agent falls back to
+bootstrapping a new cluster of its own.
+
+The agent will advertise its own address as `--cluster.advertise-address` to
+other agent nodes; if this is empty it will attempt to find a suitable address
+to advertise from a list of default network interface. The agent must be
```

Comment:
> a list of default network interface

Not sure if this is grammatically correct? If it's a list, I'd expect "interfaces"?